services/ui-src/src/utils/testing/mockFullReportJSON.tsx

```diff
@@ -698,7 +698,7 @@ export const mockFullReportJSON: ReportJson = {
             readOnlyEntityDetailsButtonText: "View",
             dashboardTitle: "Initative total count",
             countEntitiesInTitle: true,
-            tableHeader: "Initaitve name <br/> Work Plan topic",
+            tableHeader: "Initaitve name <br/> MFP Work Plan topic",
```

Comment:
Looks like there is a typo here... The beginning of this should read `Initiative`